packages/esm-cervical-cancer-app/src/cervical-cancer/cacx-visits/cacx-visits-services.component.tsx

```diff
@@ -14,32 +11,7 @@ const CaCxCervicalCancerServices: React.FC<OverviewListProps> = ({ patientUuid }
   const config = useConfig();
   const tabs = getMenuItemTabConfiguration(cacxConfigSchema, config);
 
-  return (
-    <div className={styles.tabContainer}>
-      <Tabs>
-        <TabList contained>
-          {tabs.map((tab) => (
-            <Tab key={tab.name}>{tab.name}</Tab>
-          ))}
-        </TabList>
-        <TabPanels>
-          {tabs.map((tab) => (
-            <TabPanel>
-              <EncounterList
-                patientUuid={patientUuid}
-                formList={tab.formList}
-                columns={tab.columns}
-                encounterType={tab.encounterType}
-                launchOptions={tab.launchOptions}
-                headerTitle={tab.headerTitle}
-                description={tab.description}
-              />
-            </TabPanel>
-          ))}
-        </TabPanels>
-      </Tabs>
-    </div>
-  );
+  return <TabsComponent patientUuid={patientUuid} tabsConfig={tabs} />;
```

Comment:
We should pass both configs to this component and handle the manipulation in the commons lib